yapftests/reformatter_style_config_test.py

```diff
@@ -76,6 +76,47 @@ def testOperatorStyle(self):
       style.SetGlobalStyle(style.CreatePEP8Style())
       style.DEFAULT_STYLE = self.current_style
 
+  def testOperatorPrecedenceStyle(self):
```

Comment:
Does your patch work with multiple operators like this:

```
a = (1 * 2 * 3) + (4 * 5 * 6 * 7 * 8)
```

Might be worth adding test for it.

Reply:
Indeed it does. Added your suggestion in the test 👍

Reply:
Cool! Thanks.